plugins.json

```diff
@@ -3782,6 +3782,177 @@
     },
     "updated_at": "2019-06-11T19:41:07Z"
   },
+  {
+      "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+      "icon_data": "",
+      "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-calls-v0.12.1.tar.gz",
+      "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.12.1",
+      "hosting": "",
+      "author_type": "mattermost",
+      "release_stage": "production",
+      "enterprise": false,
+      "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmPAd94ACgkQ0bVLR6XO/sRRJQ/9HR04IVavRTf6U05Oa168RSEuCevxOzd4KetCe18MS2t2w3SO9BV5vIdI4fJgI2yh1OJ6dF1FdhF+uYqfoXee8OEpv7NHqVemOPGst/QzvIGf/trV4aZCa19iE7Kjp0OdA2+w/wDScccIzh6asVW2mAgyIuVIQfuirYZQUxtDbQ61JAfEbIwmdEPn3fRztUoB2kRs2718J+WM6oTACh4KMthZRB9bq4yjEPzWKlMvGhRvQLRd/NhhS0dkuTslF8ssO4zFgYd/J2LMwoWHohUGKwpL9eHhZzKk1VkDg6OpYFVqM4jkTNDTbhq8EcTRNk17kbFb6A/VHyaySOnKyS5mpvGV8qIsrJD9AozngcKplUQNMn+ntdGEZm8EbtHyQA5vfCaMSg7dUom+VFlFiN0lX2O8Nip+xZ74aD9knfsXgMxpAeJDcV95gp9Ag9XAmElJF0biEBJPmzx2cvhm3i8DtXtKukrf3RDKh2kkv3xjzBCqZH7f2+FMftugjB5PWKkxrLj5CtJIyNSpgLOlYSFrz3trxcGdCcjITB2OogLquo0ni92TTiubIlWuYNmgXsRN0J9xAAU3OjbkaPiWDUWd+nBNEUubgOUhqAFGKAAtCFYlmeSfETlVZZRqak1UuCBaf3imFQZgCzuFZEaUfaez/HtRR9JfC0cWJ7eRFL1CDeU=",
+      "repo_name": "mattermost-plugin-calls",
+      "manifest": {
+          "id": "com.mattermost.calls",
+          "name": "Calls",
+          "description": "Integrates real-time voice communication in Mattermost",
+          "homepage_url": "https://github.com/mattermost/mattermost-plugin-calls/",
+          "support_url": "https://github.com/mattermost/mattermost-plugin-calls/issues",
+          "release_notes_url": "https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.12.1",
+          "version": "0.12.1",
+          "min_server_version": "7.6.0",
```

Comment:
Oh well I forgot to alter this :grimacing:

Reply:
yah, it won't matter. :)